index.html

```diff
@@ -3007,11 +3007,13 @@ <h3>Expiration</h3>
       </section>
 
       <section>
-        <h3>Revocation</h3>
+        <h3>Status Revocation</h3>
 
         <p>
-If revocation instructions are present, the <a>credential</a> MUST NOT
-have been revoked.
+If the <code>credentialStatus</code> propery is available the status of the 
+claim MUST be in good standing according to the <code>credentialStatus</code> 
+<a>type</a> definition for the <a>credential</a>. For example, a <a>verifier</a>
+can ensure that the status hasn't been withdrawn for cause by the <a>issuer</a>.
```

Comment:
Suggest rewriting to "ensure that the credential has not been suspended or revoked by the issuer". I think we should be more direct about what this property is about since it's caused confusion by being too generic about it before.

Reply:
I didn't want to over-specify status and get in the way of definitions that might be introduced and accepted in the "credentialStatus type" registry - it's so open world right now, we have to be careful.  I'm not sure how I would make it better.

Reply:
@msporny any suggestions here?  Is it wrong, confusing, or "ok, but could be better" - I'd like to merge and move on if it's "ok, but could be better".

Reply:
Ok, but could be better. :)